File: src/main/java/net/kenro/ji/jin/purescript/psi/impl/PSStringLiteralImpl.java
package net.kenro.ji.jin.purescript.psi.impl;

import com.intellij.lang.ASTNode;
import com.intellij.psi.PsiElementVisitor;
import net.kenro.ji.jin.purescript.psi.PSStringLiteral;
import net.kenro.ji.jin.purescript.psi.PSVisitor;
import org.jetbrains.annotations.NotNull;

public class PSStringLiteralImpl extends PSPsiElement implements PSStringLiteral {

    public PSStringLiteralImpl(final ASTNode node) {
        super(node);
    }

}

File: src/main/java/net/kenro/ji/jin/purescript/psi/impl/PSStringLiteralImpl.kt
package net.kenro.ji.jin.purescript.psi.impl

import com.intellij.lang.ASTNode
import net.kenro.ji.jin.purescript.psi.PSStringLiteral

class PSStringLiteralImpl(node: ASTNode) : PSPsiElement(node), PSStringLiteral